packages/experimental/RadioGroup/src/Radio/Radio.styling.ts

```diff
@@ -52,11 +53,11 @@ export const stylingSettings: UseStylingOptions<RadioProps, RadioSlotProps, Radi
           height: tokens.radioInnerCircleSize,
           width: tokens.radioInnerCircleSize,
           backgroundColor: tokens.radioFill,
-          left: 4,
-          top: 4,
+          left: tokens.radioInnerCircleLeft,
+          top: tokens.radioInnerCircleTop,
```

Comment:
Any reason we can't use margin/padding? The left/right/top style props always confused me..